set-web/src/main/java/eu/europeana/set/web/service/impl/UserSetServiceImpl.java

```diff
@@ -493,6 +493,27 @@ public UserSet fetchDynamicSetItems(UserSet userSet, String apiKey, String sort,
 	}
     }
 
+    @Override
+    public UserSet fetchDynamicSetItemDescriptions(UserSet userSet, String apiKey, String sort, String sortOrder, int pageNr,
+	    int pageSize) throws HttpException {
+
+	String url = buildSearchApiUrl(userSet, apiKey, sort, sortOrder, pageNr, pageSize);
+
+	SearchApiResponse apiResult;
+	try {
+	    apiResult = getSearchApiClient().searchItemDescriptions(url, apiKey, "UserSet.FETCH_ITEMS");
```

Comment:
what is the meaning of UserSet.FETCH_ITEMS and what is its use?

Reply:
FETCH_ITEMS is just action description for response. I did it simalar as it was in existing method searchItems